src/mlpack/methods/pca/pca.hpp

```diff
@@ -40,8 +40,8 @@ class PCAType
    *
    * @param scaleData Whether or not to scale the data.
```

Comment:
Should we remove the note from the file ``Note this class will be changed to have the name PCA in mlpack 3.0.0``?